### PR TITLE
Change import now to use webhook instead

### PR DESCRIPTION
**Pull request by tabfugnic**
* Webhook url will take place of import now button
* Import now tests have been removed

### CODE REVIEW

File: spec/models/icims/connection_spec.rb [hunk 1]
```diff
@@ -2,7 +2,10 @@
 
 describe Icims::Connection do
   describe "associations" do
+    subject { build(:icims_connection) }
     it { should belong_to(:user) }
+    it { should validate_presence_of(:user_id) }
+    it { should validate_uniqueness_of(:api_key) }
```
Comment (hrs) [inline spec/models/icims/connection_spec.rb]: `expect` syntax?

File: config/routes.rb [hunk 2]
```diff
@@ -8,7 +8,10 @@
   resources :jobvite_imports, only: [:create]
   resources :icims_imports, only: [:create]
 
-  resources :icims_candidate_imports, only: [:create]
+  post(
+    "/icims_candidate_imports/:api_key" => "icims_candidate_imports#create",
+    as: :icims_candidate_imports
```
Comment (hrs) [inline config/routes.rb]: TRAILING COMMA POLICE :police_car: :cop: :oncoming_police_car:

File: app/models/icims/connection.rb [hunk 3]
```diff
@@ -30,6 +31,10 @@ def required_namely_field
       AttributeMapper.new.namely_identifier_field.to_s
     end
 
+    def set_api_key
+      self.api_key = SecureRandom.hex(10)
```
Comment (hrs) [inline app/models/icims/connection.rb]: So when you create a connection, you generate a new API key. This is the key that the iCIMS webhook is expected to use to connect, right?

**Comment by hrs** [comment]
Couple bits of minutia, but otherwise you're lookin' fine.

File: app/models/icims/connection.rb (hunk 3 above)
Comment (tabfugnic) [inline app/models/icims/connection.rb]: That is correct. Thoughts?

Comment (hrs) [inline app/models/icims/connection.rb]: Just making sure I understood. I might make it 30 characters or so, because there's no reason not to.

**Pull request merged**